test/tests/activitiesTest.js

```diff
@@ -187,5 +238,15 @@ module.exports = function (request, db) {
                         });
                 });
         });
+
+        it('should DELETE an activity', function (done) {
+            request.delete('/api/activities/' + idActivity)
+                .expect(200)
+                .set('X-Gleaner-User', 'Teacher1')
+                .end(function (err, res) {
+                    should.not.exist(err);
+                    done();
```

Comment:
How do we confirm that it has been removed? Also, I'll need more in-depth information about where and how are the PERMISSIONS stored. On the other hand, elasticsearch is really out of my knowledge so we'll have to have a private lesson. Thanks :)